pandapower/converter/powermodels/to_pm.py

```diff
@@ -307,17 +310,17 @@ def ppc_to_pm(net, ppci):
         branch = dict()
         branch["index"] = idx
         branch["transformer"] = bool(idx > n_lines)
-        branch["br_r"] = row[BR_R].real
-        branch["br_x"] = row[BR_X].real
-        branch["g_fr"] = - row[BR_B].imag / 2.0
-        branch["g_to"] = - row[BR_B].imag / 2.0
-        branch["b_fr"] = row[BR_B].real / 2.0
-        branch["b_to"] = row[BR_B].real / 2.0
+        branch["br_r"] = row[BR_R].real / baseMVA
```

Comment:
I think the following is correct:

z_base_ohm = square(v_base_kv) / baseMVA
br_r = br_r / z_base_ohm

Plase add a test to catch the incorrect implementation (one way or the other) e.g. by comparing the results for the same grid but with different sn_mva.

Reply:
I think I misunderstood the code.
BR_R, BR_X are already in p.u. in ppc for baseMVA in ppc, and here the values are adjusted to match the sn_mva value of 1 MVA?

Reply:
So in ppc the following happens: R_Ohm / ((v_base_kv)**2 / baseMVA) 

Then in your code you divide by baseMVA to correct to 1 MVA.

Is that correct?

Reply:
hi,

I down these modifications by comparing the grid model in PowerModels (PM) with different baseMVA.  The default baseMVA in PM is 1.0, so I transform the grid data in pandapower according to sn_mva=1.0 before it is used in PM.
In pandapower, with sn_mva==1, the BR_R is 0.002, and with sn_mva==100, the BR_R is 0.2. 

I have just added a test for ac_opf with a different sn_mva. The results locally are the same. Lets the see the official tests

Reply:
Thank you for considering my comments!